Yoast/javascript
# Fix the text analysis breaking on the word ying

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where the text analysis would break if the text contains the word "Ying".

## Test instructions

This PR can be tested by following these steps:

* To reproduce
  * Check out `develop` in `javascript` and in `premium-configuration`
  * Start your `content-analysis` app
  * Create a text that contains the word `Ying` and add a keyphrase, set your locale to English
  * See that a number of keyphrase-based assessments break
* Check that it is fixed now
  * Check out `fix-ying-stemming` in `javascript`
  * Refresh your `content-analysis` app
  * See that the text analysis returns good results and the console error disappeared

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://github.com/Yoast/bugreports/issues/932
